build.mill

```diff
@@ -26,6 +28,7 @@ object Deps {
   val sourcecode = ivy"com.lihaoyi::sourcecode::0.4.2"
   val utest = ivy"com.lihaoyi::utest::0.8.4"
   val expecty = ivy"com.eed3si9n.expecty::expecty::0.16.0"
+  val collectionCompact = ivy"org.scala-lang.modules::scala-collection-compat:2.9.0"
```

Comment:
Why is this needed?

Reply:
Scala 2.12 can't import `scala.jdk.CollectionConverters` without this.

Reply:
It seems have a higher version 2.12.0, let me update it.